README.md

```diff
@@ -20,27 +22,36 @@ Currently, it has an LSP component and a corresponding VS Code(ium) extension.
 ## Development guide
 
 ### Prerequisites
+
 Install:
+
 - [Rust](https://www.rust-lang.org/) for the LSP itself
 - [node](https://nodejs.org/en) for the VS Code extension; it may be easiest to
     install via [nvm](https://github.com/nvm-sh/nvm)
 
 ### First time setup
+
 1. Clone this repository locally
 2. Open it in VS Code; it's needed to run the extension
 3. In the `addons/vscode` subdirectory:
     1. Run `npm install` to install extension dependencies
     2. Run `npm run compile` to build the extension
 4. Run through the development cycle once to initialize and test everything
+5. If you want to install the dev version of the extension: After finishing the
+last step in Development Cycle, then press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>,
+and choose `Developer: Install Extension from Location...` and choose the
+the root directory for the extension. There will not be any messages, but
```

Comment:
```suggestion
the directory for the extension: `addons/vscode/`. There will not be any messages, but
```

This wasn't clear to me, I kept selecting the root of the whole repo 😅

Reply:
Oh you're right. That's what I actually did. Ups.